README.md

```diff
@@ -70,16 +70,16 @@
 python3 manage.py createsuperuser --username admin
 ```
 
-**Setup is now complete!** :tada:
+**Setup is now complete!** :tadaa:
```

Comment:
`:tada:` is the code for the :tada: emoji, not a typo :smile:

Reply:
oops